tests/FakeItEasy.Analyzer.VisualBasic.Tests/FakeItEasy.Analyzer.VisualBasic.Tests.csproj

```diff
@@ -0,0 +1,179 @@
+      <HintPath>..\..\packages\xunit.abstractions.2.0.0\lib\net35\xunit.abstractions.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="xunit.core, Version=2.1.0.3179, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\xunit.extensibility.core.2.1.0\lib\dotnet\xunit.core.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <Reference Include="xunit.execution.desktop, Version=2.1.0.3179, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\xunit.extensibility.execution.2.1.0\lib\net45\xunit.execution.desktop.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\FakeItEasy.Analyzer.Tests\Helpers\CodeFixVerifier.cs">
+      <Link>Helpers\CodeFixVerifier.cs</Link>
+    </Compile>
+    <Compile Include="..\FakeItEasy.Analyzer.Tests\Helpers\CodeFixVerifier.Helper.cs">
+      <Link>Helpers\CodeFixVerifier.Helper.cs</Link>
+    </Compile>
+    <Compile Include="..\FakeItEasy.Analyzer.Tests\Helpers\DiagnosticResult.cs">
+      <Link>Helpers\DiagnosticResult.cs</Link>
+    </Compile>
+    <Compile Include="..\FakeItEasy.Analyzer.Tests\Helpers\DiagnosticVerifier.cs">
+      <Link>Helpers\DiagnosticVerifier.cs</Link>
+    </Compile>
+    <Compile Include="..\FakeItEasy.Analyzer.Tests\Helpers\DiagnosticVerifier.Helper.cs">
+      <Link>Helpers\DiagnosticVerifier.Helper.cs</Link>
+    </Compile>
+    <Compile Include="NonVirtualSetupAnalyzerTests.cs" />
+    <Compile Include="UnusedReturnValueAnalyzerTests.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config">
+      <SubType>Designer</SubType>
+    </None>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\FakeItEasy.Analyzer.VisualBasic\FakeItEasy.Analyzer.VisualBasic.csproj">
+      <Project>{7BAEB7DC-635D-425B-94AB-826AB65FBBD7}</Project>
+      <Name>FakeItEasy.Analyzer.VisualBasic</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\src\FakeItEasy\FakeItEasy.csproj">
+      <Project>{74165807-676F-4464-B607-8770C13E8991}</Project>
+      <Name>FakeItEasy</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\FakeItEasy.Tests\FakeItEasy.Tests.csproj">
+      <Project>{0E26272D-045A-47C2-8928-BD9B980115D5}</Project>
+      <Name>FakeItEasy.Tests</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
+  </ItemGroup>
+  <ItemGroup>
+    <Analyzer Include="..\..\packages\Microsoft.CodeAnalysis.Analyzers.1.1.0\analyzers\dotnet\cs\Microsoft.CodeAnalysis.Analyzers.dll" />
+    <Analyzer Include="..\..\packages\Microsoft.CodeAnalysis.Analyzers.1.1.0\analyzers\dotnet\cs\Microsoft.CodeAnalysis.CSharp.Analyzers.dll" />

```

Comment:
This is not a mistake. Even though the project tests a VB.NET analyzer, the code is in C#, so it needs the C# analyzer. Actually, I'm not sure this analyzer (which is a "meta-analyzer" to ensure the analyzer is written correctly) is necessary, since this project doesn't *implement* an analyzer... I think it was there in the project template, and I just copied and edited the C# project. I'll see if it can be removed.